internal/api/response.go

```diff
@@ -41,8 +41,9 @@ func RespondWithJSON(w http.ResponseWriter, code int, payload interface{}) {
 	w.Write(response)
```

Comment:
This line courses errcheck. We should check the error from Write() function but RespondWithJson() has referenced a lot of places. It will be a huge change. Before fixing the issue I need confirmation to do it.

Reply:
Please do not change it now. I want to think about it.

Reply:
Ok, I will not change it